### DISCUSSION TITLE
remote Receiver abuse

### DISCUSSION THREAD

**Discussion by bminish** [rrobinett/wsprdaemon#16] [General]
Someone is running "wsprdaemon_v2.10j" on 3 out of 4 of the available public instances on my kiwi sdr. 
I don't mind one on occasion, but using up all the slots is rude and denies other users of airtime.

what happens if multiple users start running this project, will we see all the better receivers choked up gathering essentially the same decodes ?

**Comment by rrobinett** [comment]
On the Admin->Network page of your Kiwis you can add the intruder's IP
address to the blacklist.

On Tue, Aug 3, 2021 at 2:39 PM Brendan Minish ***@***.***>
wrote:

> Someone is running "wsprdaemon_v2.10j" on 3 out of 4 of the available
> public instances on my kiwi sdr.
> I don't mind one on occasion, but using up all the slots is rude and
> denies other users of airtime.
>
> what happens if multiple users start running this project, will we see all
> the better receivers choked up gathering essentially the same decodes ?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rrobinett/wsprdaemon/discussions/16>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIFAQZELP362BRVSLXFJYVTT3BOY3ANCNFSM5BPVP7OA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Rob Robinett
AI6VN
***@***.***
mobile: +1 650 218 8896

**Reply by bminish** [reply]
Doesn't help me much if the abuser is dynamic & in any case I'm probably ok with one instance (& perhaps more if someone gets in touch and convinces me that they are doing something interesting )

**Comment by rrobinett** [comment]
Those of us who operate public Kiwi sites like KPH, KFS or my AI6VN/KH6
have had to configure long blacklists to keep the Asian bots off our
systems.
You can also set daily usage time limits and offer passwords to those you
want to have unrestricted access to your Kiwis.
What is your site?

On Tue, Aug 3, 2021 at 2:45 PM Brendan Minish ***@***.***>
wrote:

> Doesn't help me much if the abuser is dynamic & in any case I'm probably
> ok with one instance (& perhaps more if someone gets in touch and convinces
> me that they are doing something interesting )
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rrobinett/wsprdaemon/discussions/16#discussioncomment-1127684>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIFAQZCTMPPAZGLP7ZRE45TT3BPQ7ANCNFSM5BPVP7OA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Rob Robinett
AI6VN
***@***.***
mobile: +1 650 218 8896

**Reply by bminish** [reply]
This one is the first 'bot' I have observed. 
Yes I do have daily time limits & passwords set, but there's a balance to be struck here as I do genuinely wish to offer a decent amount of airtime to humans who might like to listen to shortwave.
